scripts/install/install-pocs.sh

```diff
@@ -92,6 +93,16 @@ case ${key} in
 esac
 done
 
+if ! ${DEVELOPER}; then
+    echo "WARNING "developer" mode not detected. You can exit and install in "developer" mode with --developer or continue."
+    select yn in "Yes" "No"; do
+        case ${yn} in
+            Yes ) echo "Proceeding with existing installation settings"; break;;
+            No ) echo "Exiting"; exit 1;;
+        esac
+    done
```

Comment:
Mostly I meant to copy the structure for asking a yes/no question in make_directories.  Here it's a bit different.

This doesn't have to be a warning and that would scare the people who are installing this on a unit (i.e. not in developer mode).

We also don't need to exit and start over, we can just set the variable based on their answer. `make_directories` was different because they would presumably have to go deal with the directory before installing.

```suggestion
    echo -n "Are you installing POCS as a developer? (for PANOPTES units, select No)"
    select yn in "Yes" "No"; do
        case ${yn} in
            Yes ) echo "Enabling developer mode. Note that you will need your GitHub username to proceed"; DEVELOPER=true; break;;
            No ) echo "Installing POCS in production mode"; break;;
        esac
    done
```